fastjsonschema/__init__.py

```diff
@@ -85,3 +87,56 @@ def compile(definition, handlers={}):
     global_state = code_generator.global_state
     exec(code_generator.func_code, global_state)
     return global_state[name]
+
+def write_code(filename, definition, handlers={}, overwrite=False):
```

Comment:
Maybe this should be `compile_to_code` which produces only string and user can do whatever they want with it. Maybe we could even provide `__main__` so it can be used from command line tool. Something like `python -m fastjsonschema "{...}"` and it would produce code to the console and user can pipe it to whether they like. :-)

Also would be awesome to write down version of the library. In case of bug or something will user be able to tell with which version it was generated. This is little bit tricky but I use this solution for my projects: https://github.com/horejsek/python-sqlpuzzle/blob/master/setup.py#L8

I will do that!